stable/grafana/templates/configmap.yaml

```diff
@@ -1,15 +1,52 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  name: {{ template "grafana.fullname" . }}
   labels:
-    app: {{ template "grafana.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    component: "{{ .Values.server.name }}"
-    heritage: "{{ .Release.Service }}"
-    release: "{{ .Release.Name }}"
-  name: {{ template "grafana.server.fullname" . }}-config
+    app: {{ template "grafana.name" . }}
+    chart: {{ template "grafana.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
 data:
-  {{- if .Values.server.installPlugins }}
-  grafana-install-plugins: {{ .Values.server.installPlugins | quote }}
+{{- if .Values.plugins }}
+  plugins: {{ default "" .Values.plugins | quote }}
```

Comment:
Don't use the `default` function. Default values should come from `values.yaml`. Apply everywhere.

Reply:
fixed.